projects/dsp-ui/src/lib/action/pipes/string-transformation/linkify.pipe.ts

```diff
@@ -0,0 +1,58 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+/**
+ * This pipe analyses a string and converts any url into a href tags
+ */
+@Pipe({
+    name: 'dspLinkify'
+})
+export class LinkifyPipe implements PipeTransform {
+
+    // constructor(private _domSanitizer: DomSanitizer) { }
+
+    transform(value: string): string {
+        let stylizedText: string = '';
+        if (value && value.length > 0) {
+            for (let str of value.split(' ')) {
+                // if string/url ends with a full stop '.' or column ':' the pipe will not recognize the url
```

Comment:
This comment should be updated to align with line 19. Also, ':' is called a colon in english :)

Reply:
oh yes... colon, not column 😉 
fixed in 29c25eb

Reply:
and in ab088df